pkg/controller/add_cluster.go

```diff
@@ -26,67 +26,16 @@ import (
 
 	myspec "github.com/m3db/m3db-operator/pkg/apis/m3dboperator/v1alpha1"
 	"github.com/m3db/m3db-operator/pkg/k8sops"
-	"github.com/m3db/m3db-operator/pkg/m3admin"
 	"github.com/m3db/m3db-operator/pkg/util/eventer"
 
-	"github.com/m3db/m3/src/cluster/generated/proto/placementpb"
-	"github.com/m3db/m3/src/query/generated/proto/admin"
-
 	corev1 "k8s.io/api/core/v1"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
-
-	pkgerrors "github.com/pkg/errors"
-	"go.uber.org/zap"
-)
-
-const (
-	// defaults for placement init request
-	_defaultM3DBPort = 9000
-	_defaultZone     = "embedded"
 )
 
 var (
 	errEmptyConfigMap = errors.New("ConfigMapName cannot be empty if non-nil")
 )
 
-// EnsurePlacement ensures that a placement exists otherwise create one
```

Comment:
I assume this code was replaced beforehand, didn't see it replaced within this PR.

Reply:
Yeah turns out I couldn't find it used anywhere